Makefile

```diff
@@ -47,7 +49,7 @@ ifeq ($(USE_IMAGE_DIGESTS), true)
 endif
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= quay.io/openstack-k8s-operators/cinder-operator:latest
```

Comment:
So this is actually flagged in some of our build automation. https://github.com/openstack-k8s-operators/cinder-operator/blob/master/.github/create_bundle.sh#L50 The same default exists in other operators

Reply:
Is there a danger of a local build pushing this to quay if the user happens to have write permission?

Reply:
There is, and I had a previos version of the patch where it checked in  the `docker-push` target to prevent it from working if it was trying to push there, but I thought that we could be using it on the GH actions to actually push there...

Reply:
GH action already does - https://github.com/openstack-k8s-operators/cinder-operator/actions/runs/3112473499/jobs/5045972880. On the main repo the action will push to quay.io/openstack-k8s-operators, on a fork it will push to quay.io/<quay_username>

Reply:
Oh, but it doesn't use `make docker-push`, so I can add checks in our `Makefile` to prevent it!
Thanks for the reference.  :-)

Reply:
Done